espaloma/app/experiment.py

```diff
@@ -101,6 +146,7 @@ def __init__(
         data,
         states,
         metrics=[esp.metrics.TypingCrossEntropy()],
+        normalize=esp.data.normalize.ESOL100LogNormalNormalize,
```

Comment:
Could we add a comment here about what contexts will cause this normalization to be applied? I imagine this isn't being applied to the listed metric (cross-entropy loss), and only kicks in for other metrics (those for energy regression, parameter regression).